src/parser.jl

```diff
@@ -545,7 +545,11 @@ end
 #
 # flisp: parse-eq
 function parse_eq(ps::ParseState)
-    parse_assignment(ps, parse_comma)
+    if peek(ps) == K"export"
+        parse_comma(ps)
+    else
+        parse_assignment(ps, parse_comma)
+    end
```

Comment:
I think because `export scoped=true a, b` parses as `(export scoped) = (true a, b)` without it.

Most folk seem to prefer `public a, b` instead, so it will no longer be necessary.